springfox-schema/src/main/java/springfox/documentation/schema/property/OptimizedModelPropertiesProvider.java

```diff
@@ -218,6 +220,58 @@ public boolean apply(Annotation input) {
 
   }
 
+  private boolean shouldBeIncludeDueToJsonView(BeanPropertyDefinition jacksonProperty,
+                                               ModelContext givenContext) {
+
+    Class classToRestrictOn = null;
+    JsonView jsonView = givenContext.getJsonView();
+    if (jsonView != null) {
+      classToRestrictOn = jsonView.value()[0];
```

Comment:
The [0] is because spring does not support more than one class in the jsonView declared in the method of the controller